nengo_viz/components/value.py

```diff
@@ -1,6 +1,7 @@
 import nengo
 import numpy as np
 import struct
+import collections
```

Comment:
<grins> I was just looking at exactly that and thinking I should fix it, but then thinking "meh, no one's going to be that anal about it...."  ;)

I'll move it (and the struct import) above and add the blank line.  :)